components/atom/card/src/index.scss

```diff
@@ -0,0 +1,34 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$c-atom-card-highlight: #fffed4 !default;
+
+.sui-AtomCard {
+  display: flex;
+
+  &-media {
+    width: 33%;
```

Comment:
this might be a component own variable so everyone could override it as wanted

Reply:
Fixed!